generate_so.py

```diff
@@ -649,4 +650,35 @@ def _make(dirpath: str) -> None:
         version='0.1',
         create_date='2019-07-21T12:55:20Z',
     )
+
+    # TSOP
+
+    _make('out/tsop/pkg')
+    generate_pkg(
+        dirpath='out/tsop/pkg',
+        author='Tubbles',
+        # Name extrapolated from to IPC7351C
+        name='TSOP{pin_count}P{pitch}_{body_length}X{lead_span}X{height}L{lead_length}X{lead_width}',
+        description='{pin_count}-pin Thin small outline package (TSOP), '
```

Comment:
Nitpick: Can you capitalize the name? "Thin Small Outline Package"

Reply:
Fixed